presets/android-dev.sh

```
@@ -0,0 +1,3 @@
+PRESET_NAME="Android Development"
+PRESET_DESCRIPTION="Android development environment"
+PRESET_PACKAGES=("python" "nodejs-lts" "neovim" "git" "fzf" "bat" "eza" "nala" "open-ssh")
```

Shellcheck (0.11.0)

[warning] PRESET_PACKAGES appears unused. Verify use (or export if used externally).

(SC2034)